demo/EXPLOR3R/auto-drive.py

```diff
@@ -48,6 +48,8 @@
 ir = InfraredSensor(); assert ir.connected
 ts = TouchSensor();    assert ts.connected
 
+print('Robot Starting')
```

Comment:
> Since this is a demo, we could use this as an opportunity to demonstrate espeak functionality, and change the line to

I can't say I'm particularly excited to talk at people every time they start his robot, but I suppose it could be a good way to help people discover this functionality.

> Is there a demo somewhere on how to vim a .sh file and chmod + x it so it can be launched? Or is there some other way to make these scripts executable on the brickman?

I believe the README has a section on preparing Python files for execution.